advisories/1.1.4/BRSA-5qpiqmz2r4pt.toml

```diff
@@ -10,11 +10,21 @@ package-name = "kernel-5.10"
 patched-version = "5.10.234"
 patched-epoch = "0"
 
+[[advisory.products]]
```

Comment:
nit: there are just two advisories that interleave the versions. I'd like to keep them all consistent.

Reply:
fixed